### ISSUE TITLE
yarn v2 complains about missing webpack dependency

### ISSUE THREAD

**Issue by lionelB** [martpie/next-transpile-modules#105]
- [x] I HAVE READ [THE FAQ](https://github.com/martpie/next-transpile-modules#faq) AND MY PROBLEM WAS NOT DESCRIBED THERE

 
**Describe the bug**
Using yarn v2 (berry) I have an error 

**To Reproduce**
- install next.js & next-transpile-module
- add transpile module to the config
- run next dev

**Expected behavior**
next running

**Setup**

- Next.js version: 9.5.2
- `next-transpile-modules` version: 4.0.2
- Node.js version: 14.3.0
- `npm`/`yarn` version: yarn 2.1.1
- Operating System: mac osx catalina
- Webpack 4 or 5: 4

It seems that the problem come from how dependencies are defined. `webpack`  is defined as a devdependency and thus require it in the plugin. maybe it should be defined as a peerDep ?

<details>
<summary>Error</summary>

```
Error: next-transpile-modules tried to access webpack, but it isn't declared in its dependencies; 
this makes the require call ambiguous and unsound. 
```
</details>

**Comment by jyunhanlin** [comment]
We have the same problem but with yarn v1.
It happened with `yarn install --prod` then `yarn build`

**Comment by Mosoc** [comment]
Actually, webpack is a dependency in main next package.

Ref: https://github.com/vercel/next.js/blob/canary/packages/next/package.json#L119

The workaround is add back  "webpack": "^4.44.1"` in dependency section in order to install wepack properly.
I think the smell of that solution is a little bit strange.

**Comment by martpie** [comment]
Thanks for the report, it's probably due to how we make the Webpack 5 detection. I will remove the auto-detection and I'll try to submit a fix soon.

**Comment by martpie** [comment]
There: as webpack is a devDeps, it's not installed when pruning/installing your deps for productions, hence the (probable) "cannot require blabla"

https://github.com/martpie/next-transpile-modules/blob/385d34395805f667c4f5b657addbb8af1ab1d89b/src/next-transpile-modules.js#L3

**Comment by martpie** [comment]
Should be fixed in `4.1.0`, can you confirm?

**Comment by lionelB** [comment]
yup! thanks
